frontend/src/components/Agreements/AgreementEditor/AgreementEditorContext.hooks.js

```diff
@@ -1,11 +1,11 @@
 import { useContext } from "react";
-import { SERVICE_REQ_TYPES } from "../../ServicesComponents/ServicesComponents.constants";
+import { AGREEMENT_TYPES, SERVICE_REQ_TYPES } from "../../ServicesComponents/ServicesComponents.constants";
 import { AgreementEditorContext, EditAgreementDispatchContext } from "./AgreementEditorContext";
 
 export const defaultState = {
     agreement: {
         id: null,
-        agreement_type: null,
+        agreement_type: AGREEMENT_TYPES.CONTRACT,
```

Comment:
looks like we're also setting a default agreement type when creating a new agreement?  good idea while we're at it...

Reply:
good defaults since that is the only Agreement Type we support(for now)